packages/models-library/src/models_library/api_schemas_webserver/auth.py

```diff
@@ -83,9 +83,10 @@ class ApiKeyGet(BaseModel):
     api_secret: str
 
     model_config = ConfigDict(
+        from_attributes=True,
```

Comment:
is this used in conjonction with the DB? if yes how was it even working before?

Reply:
This model in particular needs quite some upgrades https://github.com/ITISFoundation/osparc-simcore/issues/4959